django/utils/functional.py

```diff
@@ -20,12 +20,19 @@ class cached_property:
 
     Optional ``name`` argument allows you to make cached properties of other
     methods. (e.g.  url = cached_property(get_absolute_url, name='url') )
+
+    On Python 3.6 or more, the name argument isn't needed.
+    (e.g.  url = cached_property(get_absolute_url) )
```

Comment:
Wrap this into previous paragraph so we only have one explanation of ``name``?

```
On Python <3.6, the ``name`` argument should be passed to make a cached property from another method without a name collision, e.g. ``url = cached_property(get_absolute_url, name='url')``.
```

Reply:
done